package.json

```diff
@@ -2356,6 +2356,12 @@
 						"markdownDescription": "EXPERIMENTAL: Whether to look for custom script definitions at `dart/config/ide/flutter.json` in Bazel workspaces. Currently supported for macOS and Linux only.",
 						"scope": "window"
 					},
+					"dart.daemonPort": {
+						"type": "number",
+						"default": false,
+						"markdownDescription": "EXPERIMENTAL: Daemon port",
+						"scope": "window"
```

Comment:
If we're gong to read this via the `ResourceConfig` class, I think we should set this to `"resource"`.